src/Tribe/Aggregator/Record/Abstract.php

```diff
@@ -1286,7 +1286,7 @@ public function has_queue() {
 	}
 
 	public function get_event_count( $type = null ) {
-		if ( is_null( $type ) ) {
+		if ( $type === null ) {
```

Comment:
:no_entry_sign: Use Yoda Condition checks, you must.

<sub>hash: b24d6defb9dbda12fb5153fa5606991a</sub>